README.md

```diff
@@ -1,282 +1,584 @@
 
+  
+
 ```
+
 * : Done
+
 ```
 
+  
+
 The wildcard has priority in this integration. This means that when processing transitions, the system will first apply any wildcard rules before considering other specific transitions. So, if you have a wildcard transition and other defined transitions in your map, the wildcard transition will be executed first, and then the system will proceed with the other transitions according to the rules defined.
 
+  
+
 ### DRY_RUN
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: false**
 
+  
+
 Execute a sync but only log API calls to Jira which would create/modify Jira Issues.
 
+  
+
 ### `jira-link-id`
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: None**
 
+  
+
 **Description:** This field specifies the Jira link ID to which the newly created issue will be linked. If a value is provided, the issue will be associated with this link ID. If not specified, the issue will remain unlinked.
 
+  
+
 ### `jira-link-type`
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: 'Relates'**
 
+  
+
 **Description:** Defines the type of relationship between the new issue and the feature issue specified by the `jira-link-id`. The default relationship is 'Relates'. Other types can be used depending on your Jira configuration, such as 'Blocks', 'Is blocked by', or 'Duplicates'.
 
+  
+
 ### `include-all-products`
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: false**
 
+  
+
 **Description:** A boolean flag that determines whether findings from all products should be included in the sync process. When set to true, the tool will include findings from other products beyond those specified. When false, only findings from the specified products will be processed.
 
+  
+
 ### `skip-products`
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: None**
 
+  
+
 **Description:** A comma-separated list of product names for which findings should be excluded from the sync. This allows you to omit specific products from the synchronization process.
 
+  
+
 ### `jira-link-direction`
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: None**
 
+  
+
 **Description:** Specifies the direction of the issue link in Jira, which can be either 'inward' or 'outward'. 'Inward' means the new issue will be linked to the feature issue, while 'outward' means the feature issue will be linked to the new issue.
 
+  
+
 ### `jira-labels-config`
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: None**
 
+  
+
 **Description:** Specifies a stringified configuration for labels to be applied to the Jira issue. Labels are used to categorize and tag issues, making them easier to search and filter.
 
+  
+
 ### `jira-watchers`
 
+  
+
 **Required: No**
 
+  
+
 **Default Value: ''**
 
+  
+
 **Description:** Comma separated list of User Emails for Atlassion Jira or User EUA IDs for Enterprise Jira.
 
+  
+
 ### `jira-watchers`
```

Comment:
`jira-watchers` section is repeated with a different description. Is this intentional?

Reply:
This was fixed - jira-labels-config was replaced by jira-watchers in second occurrence